### PR TITLE
feat(InlineEdit): apply tokens

### PR DESCRIPTION
**Pull request by andrew-ronaldson**
Closes https://github.com/patternfly/patternfly/issues/5736

### CODE REVIEW

File: src/patternfly/components/InlineEdit/inline-edit.scss [hunk 1]
```diff
@@ -1,6 +1,6 @@
 // @debug $inline-edit; // check your variable names located in src/patternfly/sass-utilities/component-namespaces.scss
 
-.#{$inline-edit} {
+:root, .#{$inline-edit} {
```
Comment (mattnolting) [inline src/patternfly/components/InlineEdit/inline-edit.scss:3]: Can we use the `:where` pseudo here?

```suggestion
:root, 
:where(.#{$inline-edit}) {
```

File: src/patternfly/components/InlineEdit/inline-edit.scss [hunk 2]
```diff
@@ -114,4 +114,4 @@
   br {
     display: none;
   }
-}
+}
```
Comment (mattnolting) [inline src/patternfly/components/InlineEdit/inline-edit.scss:117]:
```suggestion
}

```

**Review by srambach — APPROVED** [review]
Just the newline! 🦉

**Comment by andrew-ronaldson** [comment]
> Just the newline! 🦉

fixed with the next commit.

**Review by mcoker — APPROVED** [review]
👑
🐸👍

**Pull request merged**